include/jot.h

```diff
@@ -76,6 +76,7 @@
 #define PR_JOT_LOGFMT_TIME_KEY		"local_time"
 #define PR_JOT_LOGFMT_UID_KEY		"uid"
 #define PR_JOT_LOGFMT_USER_KEY		"user"
+#define PR_JOT_LOGFMT_VAR_KEY		"VAR:"
```

Comment:
The Jot API didn't actually support values that used the Var API, surprisingly.  This work revealed a gap, so part of this PR is adding that support to the Jot API.